coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -1365,11 +1365,14 @@ def test(self, backend, rank, dims):
         self.run_compare_torch(input_shape, model, backend=backend)
 
 class TestConstantPad(TorchBaseTest):
-    @pytest.mark.parametrize("backend", backends)
-    def test_constant_pad_1d(self, backend):
-        input_shape = (3, 4, 5)
-        model = torch.nn.ConstantPad1d((5, 6), 3.5).eval()
-        self.run_compare_torch(input_shape, model, backend=backend)
+    @pytest.mark.parametrize(
+        "input_shape, pad, backend",
+        itertools.product([2,2], (1,1), backends)
+    )
+    def test_constant_pad_nd(self, input_shape, pad, backend):
+        input_data = torch.rand(input_shape)
+        model = torch.constant_pad_nd(input_data, pad)
+        run_compare_torch(input_shape,model,backend=backend)
```

Comment:
please update it to `self.run_compare_torch`

Reply:
sorry completely forgot about that.